app/assets/stylesheets/components/_alert.scss

```
@@ -1,6 +1,7 @@
 .usa-alert--info-important {
   @include u-bg('accent-cool-darker');
   color: color('white');
+  font-weight: 500;
```

Because we only define 400 and 700 font weights, and due to how [fallback weight behavior](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#fallback_weights) would choose the normal font weight, I wouldn't expect this change to have any effect, and in my testing it appears visually the same as if this line didn't exist at all.

If we wanted bold, I would expect...

```suggestion
  font-weight: bold;
```